src/Templates/src/templates/maui-mobile/Pages/Controls/CategoryChart.xaml

```diff
@@ -27,24 +27,44 @@
                 <chart:SfCircularChart.Legend>
                     <controls:LegendExt Placement="Right">
                         <chart:ChartLegend.LabelStyle>
-                            <chart:ChartLegendLabelStyle 
+                            <chart:ChartLegendLabelStyle
                                 TextColor="{AppThemeBinding 
                                 Light={StaticResource DarkOnLightBackground},
-                                Dark={StaticResource LightOnDarkBackground}}" 
-                                Margin="5" 
-                                FontSize="18" />
+                                Dark={StaticResource LightOnDarkBackground}}"
+                                Margin="5,0,0,0"
+                                FontSize="14"/>
```

Comment:
Is this change in font size intended or was it perhaps a test?

Reply:
@jsuarezruiz Setting the font size to 18px made it look larger and occupied more height in the legend layout. Therefore, reduced it to 14px for a more balanced and visually appealing appearance.